types/mapbox-gl/mapbox-gl-tests.ts

```diff
@@ -1098,3 +1098,9 @@ expectType<mapboxgl.Visibility>('none');
 expectType<mapboxgl.AnyLayout>({visibility: 'none'});
 expectType<mapboxgl.AnyLayout>({visibility: 'none', 'line-cap': 'round' });
 expectType<mapboxgl.AnyLayout>({visibility: 'visible', 'icon-allow-overlap': true });
+
+/*
+ * AnyPaint
+ */
+expectType<mapboxgl.AnyPaint>({ 'circle-pitch-alignment': 'map' });
```

Comment:
Do you mind creating more test cases so all new definitions are covered?

Reply:
Yes! The tests are a bit lacking. There wasn’t basically any test in place for most layers. I can improve that.

I’ll add tests for all the existing layers.